dotnetapp-current/README.md

```diff
@@ -28,14 +28,15 @@ Follow these steps to run this sample in a Linux container:
 dotnet restore
 dotnet publish -c Release -o out
 docker build -t dotnetapp .
-docker run dotnetapp Hello .NET Core from Docker
+docker run dotnetapp
 ```
 
-Follow these steps to run this sample in a  Windows container:
+Follow these steps to run this sample in a  Windows container.
+Note: Before doing the follwing, make sure you are running in "Windows Container" mode. You can do this by Right Clicking the Docker app (in the System Tray) -> "Switch to Windows Containers".
```

Comment:
Consider adding a link to https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers for additional information.